.golangci.yml

```diff
@@ -21,7 +21,6 @@
 
 run:
   concurrency: 6
-  go: 1.22.0
```

Comment:
This seems strange, is that intentional?

Reply:
Go version gets picked up via somewhere else. @jhoward-lm recommended the change.

Reply:
Sounds resolved.